DependencyInjection/Configuration.php

```diff
@@ -466,6 +466,19 @@ private function addOrmSection(ArrayNodeDefinition $node): void
                         ->end()
                         ->scalarNode('proxy_dir')->defaultValue('%kernel.cache_dir%/doctrine/orm/Proxies')->end()
                         ->scalarNode('proxy_namespace')->defaultValue('Proxies')->end()
+                        ->arrayNode('controller_resolver')
+                            ->canBeDisabled()
+                            ->children()
+                                ->booleanNode('auto_mapping')
```

Comment:
I'm not sure about this naming (or maybe just the description). What this boolean will do is disabling the automatic mapping of _all_ request attributes as a criteria. But the first lookup by primary key (attempted before the criteria lookup) is not disabled by that flag. So setting this one to `false` will still not require to add an explicit `MapEntity` on each argument explicitly (that behavior is triggered when setting `enabled: false`, not `auto_mapping: false`)

Reply:
Right! How would you formulate that in a single sentence?

Reply:
I updated to `Set to false to disable using route placeholders as lookup criteria when the primary key doesn't match the argument name`, WDYT?